app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<h2>Edit Task</h2>
+
+<%= form_with(url: list_task_path, model: @task, local: true) do |form| %>
```

Comment:
I wanted to reuse the `_form` partial for tasks here, but the form had an original url of `list_tasks_path` for adding a task to a list: 

```
list_tasks POST   /lists/:list_id/tasks(.:format)     tasks#create
```

I wanted editing a task to route to `list_task_path` instead:

```
list_task PATCH  /lists/:list_id/tasks/:id(.:format) tasks#update
```

so I duplicated the form code to change the path.

Is there a way to define 2 different urls depending on the method of the path?